vtr_flow/tasks/regression_tests/vtr_reg_strong/strong_gaussian/config/config.txt

```diff
@@ -0,0 +1,29 @@
+##############################################
+# Configuration file for running experiments
+##############################################
+
+# Path to directory of circuits to use
+circuits_dir=benchmarks/verilog
+
+# Path to directory of architectures to use
+archs_dir=arch/timing/gaussian
+
+# Add circuits to list to sweep
+circuit_list_add=stereovision3.v
+
+# Add architectures to list to sweep
+arch_list_add=x_gaussian_y_uniform.xml
+arch_list_add=x_uniform_y_gaussian.xml
+arch_list_add=x_gaussian_y_gaussian.xml
```

Comment:
Since this is testing the same non-uniform global routing I'd suggest merging with the strong_delta test. Perhaps call it `strong_global_nonunifrom`.